pkg/provider/azure_privatelinkservice_repo_test.go

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
If there's no UT originally, how about adding some or skipping this empty file?

Reply:
removed. Thanks!